packages/leancode_lint/lib/src/unawaited.dart

```diff
@@ -1,2 +1,2 @@
 /// Explicitly ignore a future.
-void unawaited(final Future<void> future) {}
+void unawaited(Future<void>? future) {}
```

Comment:
This allows for situations like `unawaited(something?.func())`. Dart std lib has done that too